resources/services/fsx/filesystems.go

```diff
@@ -0,0 +1,563 @@
+						Resolver:    schema.PathResolver("RootSquashConfiguration.NoSquashNids"),
+					},
+					{
+						Name:        "root_squash_configuration_root_squash",
+						Description: "You enable root squash by setting a user ID (UID) and group ID (GID) for the file system in the format UID:GID (for example, 365534:65534)",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("RootSquashConfiguration.RootSquash"),
+					},
+					{
+						Name:        "weekly_maintenance_start_time",
+						Description: "The preferred start time to perform weekly maintenance, formatted d:HH:MM in the UTC time zone",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_fsx_filesystem_ontap_configuration",
+				Description: "Configuration for the FSx for NetApp ONTAP file system.",
+				Resolver:    schema.PathTableResolver("OntapConfiguration"),
+				Columns: []schema.Column{
+					{
+						Name:        "filesystem_cq_id",
+						Description: "Unique CloudQuery ID of aws_fsx_filesystems table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "automatic_backup_retention_days",
+						Description: "The number of days to retain automatic backups",
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "daily_automatic_backup_start_time",
+						Description: "A recurring daily time, in the format HH:MM",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "deployment_type",
+						Description: "Specifies the FSx for ONTAP file system deployment type in use in the file system.  * MULTI_AZ_1 - (Default) A high availability file system configured for Multi-AZ redundancy to tolerate temporary Availability Zone (AZ) unavailability.  * SINGLE_AZ_1 - A file system configured for Single-AZ redundancy.  For information about the use cases for Multi-AZ and Single-AZ deployments, refer to Choosing Multi-AZ or Single-AZ file system deployment (https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/high-availability-multiAZ.html).",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "disk_iops_configuration_iops",
+						Description: "The total number of SSD IOPS provisioned for the file system.",
+						Type:        schema.TypeBigInt,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Iops"),
+					},
+					{
+						Name:        "disk_iops_configuration_mode",
+						Description: "Specifies whether the number of IOPS for the file system is using the system default (AUTOMATIC) or was provisioned by the customer (USER_PROVISIONED).",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Mode"),
+					},
+					{
+						Name:        "endpoint_ip_address_range",
+						Description: "(Multi-AZ only) The IP address range in which the endpoints to access your file system are created",
+						Type:        schema.TypeString,
```

Comment:
cidr?

Reply:
can't verify what the format is, so I would keep it as a string for now